config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "mainnet": {
+    "organic": {
+      "sample_size": 3,
```

Comment:
would that make sense that the sample size of the organic is defined by the requester instead of the miner?

Reply:
we can definitely make a ticket for that - my concern is that there would need to be a hard-coded limit on the organic sample size, otherwise someone could spam the entire network and request 240 miners / call